legacy/README.md

```
@@ -51,6 +51,41 @@ cmake -S <path-to-source> -B <path-to-build> -C <path-to-source>/FairSoftConfig.
+Both packages are already added in the updated setup script for macOS.
+If found the packages from the homebrew installation directory will be used.
+
+The last problem is related to the macOS, compiler and SDK versions, such
+that it depends on the personal setup. As described in more detail at
+[macOS SDK](advanced.md#macos-sdk)! ROOT is very picky about the compiler
+and the connected SDK. Compiling older ROOT versions with newer compilers
+may need using an older SDK version. If not specified explicitly the
+latest SDK version is used. To use an older SDK version on needs to add the
+following parameter when running CMake
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)